python/trezorlib/binance.py

```diff
@@ -0,0 +1,114 @@
+from . import messages
+from .tools import expect, session
+
+
+@expect(messages.BinanceAddress, field="address")
+def get_address(client, address_n, show_display=False):
+    return client.call(
+        messages.BinanceGetAddress(address_n=address_n, show_display=show_display)
+    )
+
+
+@expect(messages.BinancePublicKey, field="public_key")
+def get_public_key(client, address_n, show_display=False):
+    return client.call(
+        messages.BinanceGetPublicKey(address_n=address_n, show_display=show_display)
+    )
+
+
+@session
+def sign_tx(client, address_n, tx_json):
+    msg = tx_json["msgs"][0]
+    envelope = messages.BinanceSignTx(
```

Comment:
this should be replaceable by `dict_to_proto(messages.BinanceSignTx, tx_json)`
(`from .protobuf import dict_to_proto`)

similarly every `json_to_*` should be replaceable by this call, let me know if there's something that doesn't work

Reply:
replaced at all relevant places and it seems to work.